proto/reaction.proto

```diff
@@ -231,6 +233,7 @@ message CrudeComponent {
   optional bool has_derived_amount = 3;
   // If the entire crude mixture was not used, need to specify an amount.
   Amount amount = 4;
+  ProductCompound.Texture texture = 5;
```

Comment:
We should pull `Texture`'s definition outside of `ProductCompound` if it is being used in multiple places

Reply:
Done in the latest commit. Not sure if this would break backward compatibility.

Reply:
Please verify your checks on ord-data datasets before closing this thread.

Reply:
> Please verify your checks on ord-data datasets before closing this thread.

verified using `ord_schema/scripts/validate_dataset.py` for small (size < 5 MB) datasets in https://github.com/open-reaction-database/ord-data/tree/main/data